tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-daily-perf-pipeline.yml

```diff
@@ -112,34 +57,24 @@ jobs:
     - name: optional_arguments
       value: -a "-a -g $(optimizeGraph) -b $(bindInputs) $(trtEPOptionsArg) $(cudaEPOptionsArg)"
 
+    - name: image
+      value: ort-$(Build.SourceBranchName)
```

Comment:
Note that we will no longer have to manually specify a branch name in the pipeline variables. The branch you use to run the pipeline is the branch used for everything.